react-native/react/common-adapters/input.desktop.js

```diff
@@ -1,98 +1,84 @@
 import React, {Component} from '../base-react'
 import {TextField} from 'material-ui'
 import {globalStyles, globalColors} from '../styles/style-guide'
+import {styles as TextStyles} from './text'
+import materialTheme from '../styles/material-theme.desktop'
 
 // Floating label isn't very controllable, not going to make this pixel perfect
 
 export default class Input extends Component {
-  render () {
-    let inputStyle = null
-    if (this.props.multiLine) {
-      inputStyle = this.props.errorText ? styles.inputMultiWithError : styles.inputMultiNormal
-    } else {
-      inputStyle = this.props.errorText ? styles.inputWithError : styles.inputNormal
+  getChildContext () {
```

Comment:
This is required to do the theming in material ui. We're not really using it yet but when we do it's plumbed